### DISCUSSION TITLE
[Beatmap Discussions]: A possibility to reference specific sections of your review in that review

### DISCUSSION THREAD

**Discussion by Skubers** [ppy/osu#25738] [Ideas] [answered]
If you were to mod a map regularly, and there was a reocurring problem in two of the diffs, you would point out a problem in one diff, and copy the link to that item, and paste it to the other diff, so that you don't copy paste the whole paragraph.

You can't do that in reviews, because there's no link to copy from since the review is not posted yet. There should be a possibility to reference other sections of the review in the review without the need of posting it.

**Comment by bdach** [comment]
this should go to https://github.com/ppy/osu-web

**Reply by Skubers** [reply]
Correct, but there's no Discussion section, nor any place for Feature Requests, and I'm not going to submit this as a bug there just to be berated by some person who cares too much about these things

**Reply by bdach** [reply]
> nor any place for Feature Requests

that's incorrect, and any person that would "berate you" - if any - would be wrong to do so.

https://github.com/ppy/osu-web/issues?q=is%3Aissue+is%3Aopen+label%3Aproposal

**Reply by Skubers** [reply]
Coming from osu!lazer repository, I guess there's no real flow for these kind of things on osu!web one, so that normies like me can know, but that's a discussion for another time. In that case I'll submit that there with the way you provided.

osu
![image](https://github.com/ppy/osu/assets/82095491/09dcd9c2-1a7e-4a6c-bb85-dea7ca039381)

osu-web
![image](https://github.com/ppy/osu/assets/82095491/cc3acf08-bf49-42cf-a10e-86041f93fa31)

**Reply by Skubers** [reply, marked as answer]
https://github.com/ppy/osu-web/issues/10798
